test/ChildMapping-test.tsx

```diff
@@ -1,12 +1,7 @@
-let React;
-let ChildMapping;
+import React from 'react';
```

Comment:
Not really a fan of TypeScript in runtime tests precisely because runtime tests are annoying to type. The previous test did work differently by clearing modules for each test. A static import is not the same.

If you want to test the types, we should have separate test files for them.

Reply:
>  I prefer not to mock or mutate modules as possible. So it would be nice to use static imports as default.

We're doing neither here. We're resetting the module which is not yet supported with Jest and ESM. So until then, we need to re-require.

Reply:
Thank you, I have two questions.
- Why do we need to reset the module even though we haven't used mock or mutating modules?
-  Can't we reset modules with Jest even if we transpile ESM to CJS by `tsc`?